monkestation/code/modules/uplink/uplink_items/job.dm

```diff
@@ -63,3 +63,11 @@
 	pod.custom_objective = objective
 	pod.RefreshParts()
 	pod.locked = TRUE // The pod shouldn't be eligible for cloner event.
+
+/datum/uplink_item/role_restricted/syndicate_plant_gene
```

Comment:
this is silly, but can you move this further up in the file, so I can TM this without conflicts?